testing/regress/ecl/README.md

```diff
@@ -0,0 +1,200 @@
+[Action]  14. Test: parquet_types.ecl
+[Pass]  14. Pass parquet_types.ecl - W20240815-111509 (7 sec)
+[Pass]  14. URL http://127.0.0.1:8010/?Widget=WUDetailsWidget&Wuid=W20240815-111509
+[Action]  15. Test: parquet_write.ecl
+[Pass]  15. Pass parquet_write.ecl - W20240815-111517 (2 sec)
+[Pass]  15. URL http://127.0.0.1:8010/?Widget=WUDetailsWidget&Wuid=W20240815-111517
+[Action]
+    -------------------------------------------------
+    Result:
+    Passing: 15
+    Failure: 0
+    -------------------------------------------------
+    Log: /home/useR/HPCCSystems-regression/log/roxie.24-08-15-11-14-29.log
+    -------------------------------------------------
+    Elapsed time: 52 sec  (00:00:52)
+    -------------------------------------------------
+```
+## Project Description
+
+This project focuses on the development of a comprehensive test suite for the recently integrated Parquet plugin within the HPCC Systems platform. The objective is to thoroughly evaluate the plugin's functionality, performance, and robustness across different scenarios and configurations. The key deliverables include defining and implementing various test cases, fixing any identified bugs, and providing extensive documentation.
+
+The test suite will evaluate all data types supported by ECL and Arrow, as well as file operations, various compression formats, and schema handling. Additionally, the test suite will measure the plugin's performance across different HPCC components and hardware configurations, conduct stress tests to identify potential bottlenecks and bugs, and compare Parquet to other file formats used in the ecosystem, such as JSON, XML, and CSV.
+
+# Test Files
+The test suite consists of 10 main test files that test different parquet functionality and operations:
+
+- parquet_types.ecl: Tests various ECL and Arrow data types
+
+- parquet_schema.ecl: Evaluates Parquet's handling of different schemas
+
+- parquet_compress.ecl: Tests different compression algorithms
+
+- parquet_write.ecl: Validates Parquet write operations
+
+- parquet_empty.ecl: Tests behavior with empty Parquet files
+
+- parquet_corrupt.ecl: Checks handling of corrupt Parquet data
+
+- parquet_size.ecl: Compares file sizes across formats
+
+- parquet_partition.ecl: Tests partitioning in Parquet files
+
+- parquet_overwrite.ecl: Validates overwrite operations
+
+- parquet-string.ecl: Focuses on string-related operations
+
+
+## Test Suite Overview
+## Type Testing
+
+Covers 42 data types including ECL and Arrow types
+Examples: BOOLEAN, INTEGER, STRING, UNICODE, various numeric types, sets, and Arrow-specific types
+
+ ### Data Type Tests
+
+#### ECL Types
+
+- BOOLEAN: True/False values.
+- INTEGER: Signed integer values.
+- UNSIGNED: Unsigned integer values.
+- REAL: Floating-point values.
+- DECIMAL: Decimal values with specified precision.
+- STRING: Fixed-length strings.
+- DATA: Binary data converted to STRING.
+- VARSTRING: Variable-length strings.
+- QSTRING: Quick strings.
```

Comment:
I'm not sure the Q in QString stands for Quick. QStrings are a simpler encoding of a limited set of characters. I'm not quite sure what the purpose of these descriptions are. Is it to describe the type or how that type is tested? In either case I think it requires more information in some cases and a second look.